src/main/java/org/codeforamerica/messaging/services/MessageService.java

```diff
@@ -44,6 +48,10 @@ public MessageService(SmsService smsService,
         this.jobRequestScheduler = jobRequestScheduler;
     }
 
+    public void setMessageSource(MessageSource messageSource) {
+        this.messageSource = messageSource;
+    }
```

Comment:
Kinda curious why this needs to be a setter?

Reply:
Instead of autowired or constructor param

Reply:
The class has to implement the interface `MessageSourceAware` which in turn requires that the setter should be implemented.